specification/semantic_conventions.md

```diff
@@ -183,3 +185,28 @@ provides a range of lines, the second `lineno` can be provided after a colon `:`
 - ` <col>` or ` <col>:<col>` - OPTIONAL - If the runtime provides a column or column range, it MAY be provided
 after the `lineno`, separated by a space.
 - literal `)`
+
+#### PPROF Profile.proto Data Format
+
+[`Profile.proto`](https://github.com/google/pprof/tree/master/proto) is a data representation for profiling data. It is
+independent of the type of data being collected and the sampling process used to collect that data.
+Log message will contain a gzip-compressed, base64-encoded protocol buffer conforming to `profile.proto`. Each message
+contains either `allocation` or `cpu` samples determined by the data type specified for the log record.
+
+For each `allocation` and `cpu` sample:
+- label `source.event.name` of type `string` OPTIONALLY can contain the name of the event that triggered the sampling
+- label `source.event.time` of type `long` MUST be set to the time when the sample was taken
```

Comment:
Can we specify more explicitly what a `long` type is. The underlying representation can very based on the language used. Maybe use `int64` or maybe explicitly call it a signed 64-bit (or 32-bit?) integer.

Reply:
I guess we should use the `proto` types

https://developers.google.com/protocol-buffers/docs/proto#scalar

Reply:
Changed `long` to `int64` and added link to protocol buffers doc. Alternatively could use `numeric` as the code generated from proto contains setters `setNum` and `setStr` it is only important to call the correct one of these.